packages/cardpay-sdk/sdk/safes/base.ts

```diff
@@ -105,6 +111,13 @@ const safeQueryFields = `
       id
     }
   }
+  reward {
+    id
+    rewardProgramID
+    rewardee {
+      id
```

Comment:
shouldn't we also be displaying how many reward tokens you have in your reward safe, as well as what token it is?